app/src/main/java/com/epfl/neighborfood/neighborfoodandroid/models/Order.java

```diff
@@ -0,0 +1,52 @@
+package com.epfl.neighborfood.neighborfoodandroid.models;
+
+import java.io.Serializable;
+import java.util.Date;
+
+public class Order implements Serializable {
+
+    private Meal meal;
+    private Date orderDate;
+    private boolean status;
+    private String mealVendor;
+    private static final String ORDER_DELIVERED = "This order was achieved the";
+    private static final String ORDER_NOT_YET_DELIVERED = "This order is being processed";

```

Comment:
Some other attributes we will need for an Order are pickupLocation and price. I'd suggest adding those already so they can be ready to use for future features.

Reply:
Yes certainly we will need to add more features for the order activity and even the meal vendor should be a user but I made it this way for now to be able to work on past orders.